### PR TITLE
[GHCR image] feat: support for multi-arch image build support

### PR DESCRIPTION
**Pull request by Rajpratik71**
Signed-off-by: Pratik Raj <rajpratik71@gmail.com>

### CODE REVIEW

File: .github/workflows/ghcr-image-build-and-publish.yml [hunk 1]
```diff
@@ -54,6 +60,7 @@ jobs:
         uses: docker/build-push-action@v3.2.0
         with:
           context: .
+          platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name != 'pull_request' }}
```
Comment (AkihiroSuda) [inline .github/workflows/ghcr-image-build-and-publish.yml:64]: > on:
>  push:

Could you add `pull_request` so that this can be tested on the CI

Comment (Rajpratik71) [reply .github/workflows/ghcr-image-build-and-publish.yml:66]: done

**Review by fahedouch — APPROVED** [review]

**Comment by fahedouch** [comment]
Thanks @Rajpratik71

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**